axelor-sale/src/main/java/com/axelor/apps/sale/service/configurator/ConfiguratorServiceImpl.java

```diff
@@ -434,14 +463,16 @@ protected SaleOrderLine generateSaleOrderLine(
    *
    * @param jsonIndicators
    */
-  protected void cleanIndicators(JsonContext jsonIndicators) {
+  public void cleanIndicators(JsonContext jsonIndicators) {
```

Comment:
this method should stay protected since it is only called from inside the service.

Reply:
Forgot to revert some changes indeed, it was because I needed to do tests at some points 
Resolved